python/test/test_algo_bench.py

```diff
@@ -110,13 +104,9 @@ def test_single_trial_gaps():
 
 def run_single_t(arguments):
```

Comment:
is _t our convention?

Reply:
No. I think you mean "run single test" so the name should be `run_single_test`.

Reply:
Pytest only picks up functions that begin with `test` not those that end with it. https://docs.pytest.org/en/6.2.x/reference.html#confval-python_functions